platform/core.osgi/nbproject/project.xml

```diff
@@ -47,14 +47,6 @@
                         <specification-version>9.0</specification-version>
                     </run-dependency>
                 </dependency>
-                <dependency>
```

Comment:
While at [it](https://github.com/apache/incubator-netbeans/pull/1063/) I've noticed this dependency on deprecated stuff. Replacing with conditional `Class.forName`.

Something for @jglick and @sdedic to review.